harness/determined/common/storage/__init__.py

```diff
@@ -109,12 +108,11 @@ def load(self, storage_dir: str) -> None:
                 check_eq(fp.read(), self.uuid, "Unable to properly load from storage")
 
     validater = Validater()
-    with manager.store_path() as (storage_id, path):
+    with manager.store_path() as (uuid, path):
```

Comment:
storage_id?
(I suppose `import uuid` should then become clean, w/o the need for aliasing as `_uuid`)

Reply:
yes, of course.  Sorry... I had to choose between pushing what I had so you could review it vs waiting till much later for you to review something more polished.  This was definitely on my todo list.